app/controllers/v0/form1095_bs_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module V0
+  class Form1095BsController < ApplicationController
+    before_action { authorize :form1095, :access? }
+    before_action :get_1095b, only: :download
+    before_action :get_available_forms, only: :available_forms
+
+    def available_forms
+      render json: { available_forms: @available_forms_arr }
+    end
+
+    def download
+      file_name = "1095B_#{download_params}.pdf"
+      send_data @form.get_pdf, filename: file_name, type: 'application/pdf', disposition: 'attachment'
```

Comment:
Not sure on this but if you're sending data here does the related route need to be a post?

Reply:
GET request should work here since the endpoint is only returning something. No data is changed/saved in the API with this endpoint